recipes/pdftools/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.37" %}
+{% set sha256 = "634cb50bcd276dda5ab39e3602acfb0370694b61b70eb357201f2bd40240ab4e" %}
+
+package:
+  name: pdftools
```

Comment:
Any thoughts on this @conda-forge/core? Would really appreciate feedback from others on how to handle this. 😄